src/Tizen.Sensor/Tizen.Sensor/RecorderPlugins/PressureRecorder.cs

```diff
@@ -0,0 +1,39 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.Sensor
+{
+    /// <summary>
+    /// The PressureRecorder class is used to start/stop record PressureSensor events.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public sealed class PressureRecorder : SensorRecorderDataManager
```

Comment:
Do not make a subclass of SensorRecorderDataManager

Reply:
yes, we redesigned the classes in the following way:
https://wormhole.sec.samsung.net/confluence/display/SRIDTizenServiceFW/Sensor+Recorder+Class+Diagram

Reply:
please check this link for explanation regarding design
https://wormhole.sec.samsung.net/confluence/display/SRIDTizenServiceFW/Sensor+Recorder+Class+Diagram+Explanation

Reply:
I have no permission to access

Reply:
please check it now.

Reply:
Still no permission